domains/transfer-agreements/TransferAgreements.API/API/Models/Connection.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace API.Models;
+
+public class Connection
+{
+    public Guid Id { get; set; }
+    public Guid CompanyId { get; set; }
+    public string CompanyTin { get; set; }
+    public Guid OwnerId { get; set; }
```

Comment:
What is Owner here? I am not I understand how a connection between two companies can have a owner.

Reply:
Owner is the 'accepter' subject id. When two connections are made when the invite is accepted - one for the receiver and one for the sender

Reply:
I am reacting to the concept of "owning" a connection. I do not understand why there is an ownership on a connection...?

Furthermore, when you explain why, you are not listing a ownership relationship, but only that it is a connection between two companies and one is `sending` and the other is `receiving`. Why not use those terms `sending`/`receiving`? Or `inviter`/`invitee`?

Reply:
Well because with this page I'm viewing _my_ connections. But maybe I'm looking at it wrong - how do you propose to do it?

Because from a connections standpoint it does not make sense to know who the invite and invitee is